test/options/attributeSeparator/closing-bracket-position/closing-bracket-position.test.ts

```diff
@@ -0,0 +1,21 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { format } from 'prettier';
+import { plugin } from '../../../../src/index';
+
+describe('Options', () => {
+	describe('closingBracketPosition', () => {
+		test('Should closing bracket of attributes stick with last attribute\'s line', () => {
```

Comment:
Please start with a lower case letter (**s**hould), this is because tests should be read as follows: `it should ...`